lib/device-sockets/ios/notification.ts

```diff
@@ -0,0 +1,47 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class IOSNotification implements IiOSNotification {
```

Comment:
Yes, I use the interface:
https://github.com/NativeScript/nativescript-cli/pull/1107/files#diff-ec63df112e054b09c6c8a97ec07ecb6aR10, 
https://github.com/NativeScript/nativescript-cli/pull/1107/files#diff-3ea8e9be6fce663fc286a4462786a297R31